third_party/merge_zip_files.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script that zips the content of the inputs zip files under a given directory
+# structure in the output zip file. "nodir" can be passed if no top-level directory
```

Comment:
nit: this makes it impossible to put the files in a directory called "nodir". Not a big deal, but why have that hole if you can just replace that with `-` (or, even better, something based on getopt?)

Reply:
I went for "-".